instance/models/instance.py

```diff
@@ -94,6 +96,8 @@ class Instance(ValidateModelMixin, TimeStampedModel):
     STOPPED = 'stopped'
     TERMINATING = 'terminating'
 
+    ERR_PROVISIONING_FAILED = 'provisioning_failed'
```

Comment:
For this case I think being explicit is better. For example, after provisioning we will move to a READY state, but READY progress can't be either RUNNING or FAILED, so we will have to ignore the flag when the status is READY?